### PR TITLE
NLSms1894 jk

### PR DESCRIPTION
**Pull request by karljonaskarlsson**

### CODE REVIEW

File: new/NLSms1894.xml [hunk 1]
```diff
@@ -0,0 +1,507 @@
...
+                                                        ሥርዓት፡ ዘከመ፡ ኀብሩ፡ አርባዕቱ።</foreign></desc>
+                                            </decoNote>
+                                            
+                                            <decoNote type="miniature" xml:id="d10">
+                                                <locus target="#5v"></locus>
+                                                <desc><ref type="authFile" corresp="AT1005Crucifixion">Crucifixion</ref> without the Cruficied under an 
+                                                    <term key="arch">arch</term> resting on <term key="column">columns</term>. Atop the arch are 
+                                                    four <foreign xml:lang="gez">ጢር፡</foreign> <term key="bird">birds</term> and four 
+                                                    <foreign xml:lang="gez">ዱራ፡</foreign> birds. </desc>
+                                                <q n="1" xml:lang="gez">ነዋ፡ በግዑ፡ ለእግዚአብሔር፡ ዘያአ<unclear>ብ</unclear>ት፡ ኃጢአተ፡ ዓለም፡<foreign xml:lang="en">Behold the Lamb of God who takes away the sin of the world.</foreign></q>
```
Comment (thea-m) [inline new/NLSms1894.xml:363]: Translations of legends should follow `<q>` containing the legend in another `<q>` (see https://betamasaheft.eu/Guidelines/?id=decorationDescription). (Technically, some of these here seem to be captions, not legends)
I also don't think that @n will be visualised - did you see this used anywhere in this context?

**Review by thea-m — CHANGES_REQUESTED** [review]
Excellent encoding, just minor remarks/suggestions

File: new/NLSms1894.xml [hunk 2]
```diff
@@ -0,0 +1,507 @@
...
+                                ኢየሱስ፡ ክርስቶስ፡ 
+                                <hi rend="rubric">ወልደ፡ ዳዊት፡ ወልደ፡</hi> 
+                                አብርሃም፨ አብር<supplied reason="omitted">ሃ</supplied>
+                                <hi rend="rubric">ም፡ ወለዶ፡ ለይስሐቅ፡</hi> 
+                                ወይስሐቅኒ፡ ወለዶ፡ 
+                                ለያዕቆብ፡
+                            </incipit>
+                            <explicit xml:lang="gez">
+                                ወናሁ፡ 
+                                አነ፡ ሀለውኩ፡ ምስ
```
[the remark this answers is not in the document]
Comment (karljonaskarlsson) [reply new/NLSms1894.xml:78]: Ok. I thought linebreaks weren't counted in xml

File: new/NLSms1894.xml (hunk 1 above)
Comment (karljonaskarlsson) [reply new/NLSms1894.xml:363]: I followed the description of a Paris MS, but will change as you suggest :)

Comment (gnizla) [reply new/NLSms1894.xml]: Sorry, I may have been the one misleading you, I sent you the Paris record, just for showing how I had approached describing Canon Tables (which is also open to debate), This is incredibly detailed, thanks,

**Pull request merged**